spec/services/forest_liana/resources_getter_spec.rb

```diff
@@ -0,0 +1,408 @@
+      user12 = User.create(name: 'Marc')
+      user13 = User.create(name: 'Xavier')
+      user14 = User.create(name: 'Paul')
+      user15 = User.create(name: 'Mickael')
+      user16 = User.create(name: 'Mike')
+      user17 = User.create(name: 'Maxime')
+      user18 = User.create(name: 'Gertrude')
+      user19 = User.create(name: 'Monique')
+      user20 = User.create(name: 'Mia')
+      user21 = User.create(name: 'Rachid')
+      user22 = User.create(name: 'Edouard')
+      user23 = User.create(name: 'Sacha')
+      user24 = User.create(name: 'Caro')
+      user25 = User.create(name: 'Amand')
+      user26 = User.create(name: 'Nathan')
+      user27 = User.create(name: 'Noémie')
+      user28 = User.create(name: 'Robin')
+      user29 = User.create(name: 'Gaelle')
+      user30 = User.create(name: 'Isabelle')
+
+      island1 = Island.create(name: 'Skull', updated_at: Time.now - 1.years)
+      island2 = Island.create(name: 'Muerta', updated_at: Time.now - 5.years)
+      island3 = Island.create(name: 'Treasure', updated_at: Time.now)
+      island4 = Island.create(name: 'Birds', updated_at: Time.now - 7.years)
+      island5 = Island.create(name: 'Lille', updated_at: Time.now - 1.years)
+
+      tree1 = Tree.create(name: 'Lemon Tree', created_at: Time.now - 7.years, island: island1, owner: user1, cutter: user1)
+      tree2 = Tree.create(name: 'Ginger Tree', created_at: Time.now - 7.years, island: island1, owner: user2, cutter: user1)
+      tree3 = Tree.create(name: 'Apple Tree', created_at: Time.now - 5.years, island: island2, owner: user3, cutter: user1)
+      tree4 = Tree.create(name: 'Pear Tree', created_at: Time.now + 4.hours, island: island4, owner: user4, cutter: user2)
+      tree5 = Tree.create(name: 'Choco Tree', created_at: Time.now, island: island4, owner: user5, cutter: user2)
+
+      location1 = Location.create(coordinates: '12345', island: island1)
+      location2 = Location.create(coordinates: '54321', island: island2)
+      location3 = Location.create(coordinates: '43215', island: island3)
+      location4 = Location.create(coordinates: '21543', island: island4)
+      location5 = Location.create(coordinates: '32154', island: island5)
+
+      reference = Reference.create()
+    end
+
+    after(:each) do
+      User.destroy_all
+      Island.destroy_all
+      Location.destroy_all
+      Tree.destroy_all
+    end
+
+    describe 'when there are more records than the page size' do
+      describe 'when asking for the 1st page and 15 records' do
+        it 'should get only the expected records' do
+          getter = ResourcesGetter.new(User, {
+            page: { size: 15, number: 1 },
+            sort: '-id',
+            timezone: 'America/Nome'
+          })
+          getter.perform
+          records = getter.records
+          count = getter.count
+
+          assert records.count == 15
+          assert count = 30
+          assert records.first.id == 30
+          assert records.last.id == 16
```

Comment:
[TESTS] You should use rspec matchers (see example on other services tests: `spec/services/forest_liana/permissions_getter_spec.rb:44`)